pkg/trie/triedb/proof/generate.go

```diff
@@ -36,28 +36,28 @@ type (
 func (nodeHandleHash) isNodeHandle()   {}
 func (nodeHandleInline) isNodeHandle() {}
 
-type step interface {
+type generateProofStep interface {
```

Comment:
```suggestion
type genProofStep interface {
```
If you're already using short hand for the types 🤷

Reply:
I renamed all!